pycbc/events/stat.py

```diff
@@ -280,8 +305,7 @@ def single(self, trigs):
         and rescale by the fitted coefficients alpha and lambda
         """
         alphai, lambdai, thresh = self.find_fits(trigs)
-        dof = 2 * trigs['chisq_dof'] - 2
-        newsnr = events.newsnr(trigs['snr'], trigs['chisq'] / dof)
+        newsnr = self.get_newsnr(self, trigs)
```

Comment:
Landscape is moaning about this in one of my [unrelated] pull-requests (I didn't rebase), so I ask here: Why is this self.get_newsnr(self, trigs) and not just self.get_newsnr(trigs) or get_newsnr(self, trigs) (ie. not a classmethod)? Is there a bug here?